index.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module "react-native-kustomer-sdk" {
```

Comment:
``@rebythecode/react-native-kustomer-sdk``

Reply:
Not anymore